src/genericComponents/Button.tsx

```diff
@@ -7,27 +7,35 @@ const Button = (props: any) => {
   const { onClick, text, variant, ...rest } = props;
 
   return (
-    <CAGButton
-      className={`${variant}-class`}
-      onClick={onClick}
-      variant={variant}
-      {...rest}
-    >
-      {text}
-    </CAGButton>
+    <>
+      <CAGButton
+        className={`${variant}-class`}
+        onClick={onClick}
+        variant={variant}
+        {...rest}
+      >
+        {text}
+      </CAGButton>
+    </>
   );
 };
 
 const CAGButton = styled(BSButton)`
   border-radius: 25px;
   box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.29);
   font-family: ${fonts.montserrat};
-  font-size: 12px;
-  font-weight: bold;
-  letter-spacing: 70;
-  line-spacing: 15;
-  padding: 16px 24px;
   text-transform: uppercase;
+  box-shadow: 0px 0px 8px 4px #0000000d;
+  height: 40px;
+  left: 754px;
+  top: 383.0390625px;
+  border-radius: 20px;
+  padding: 12px 18px 12px 18px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 0.1em;
```

Comment:
And this seems to be a bit small

Reply:
Again just from the figma design. Can adjust or put back.